docs/AOT.md

```diff
@@ -175,19 +171,18 @@ The makefile provides the following targets:
      memory area.
   *  Once everything is setup, it invokes the compiled network model by calling the
      `resnet50` function from the `resnet50.o` object file.
- * `resnet50`: it links the user-defined `resnet50_standalone.o` and auto-generated `resnet50.o`
-  into a standalone executable file called `resnet50_standalone`
-  * `run`: it runs this standalone executable with imagenet images as inputs and outputs the
-results of the network model execution.
+ * `ResNet50Bundle`: it links the user-defined `main.o` and auto-generated `resnet50.o` into a standalone executable file called `resnet50`
+  * `RunResNet50Bundle`: it runs this standalone executable with imagenet images as inputs and outputs the results of the network model execution.
 
 ### Quantized network
-To build and run the example, you just need to execute:
-* `make run`. By default, quantized Resnet50 example is going to be executed.
+All of the aforementioned targets have quantized versions in CMakeLists.txt named `...QuantizedResNet50...`. For example, to run the quantized bundle, you just need to execute:
```

Comment:
IMO the "named `...QuantizedResNet50...`" bit is a little confusing.  I think the example `ninja` command below is clear enough.